src/shared/components/ResourceLinks/index.tsx

```diff
@@ -1,42 +1,46 @@
 import * as React from 'react';
-import { Spin, Empty } from 'antd';
+import { Empty, Skeleton } from 'antd';
 import { ResourceLink } from '@bbp/nexus-sdk';
 
 import ListItem from '../List/Item';
 import ResourceLinkItem from './ResourceLinkItem';
 import InfiniteSearch from '../List/InfiniteSearch';
 
 const ResourceLinks: React.FunctionComponent<{
-  busy: boolean;
+  firstLoad: boolean;
```

Comment:
Can you not figure out the `firstLoad` by saying `busy && links.lenght === 0`

Reply:
🐑